### ISSUE TITLE
IonicLab "lost" an app

### ISSUE THREAD

**Issue by cfjedimaster** [driftyco/ionic-lab-issues#25]
I had an app I made with IonicLab and it showed up in the left hand nav. I just closed and reopened Lab and it is gone now. Obviously it still exists, but something killed it from the left hand nav.

**Comment by jbavari** [comment]
@cfjedimaster do you know if the app crashed and you re-opened it? Hmm. Trying to figure out if it got dropped some how.

When you hover over the apps, a little 'x' appears. However, that prompts a modal asking you to confirm removing it. I'm sure that didnt show.

**Comment by cfjedimaster** [comment]
Nope, I didn't do that. I was testing the heck out if - specifically
messing with Serve and Build and stuff, so maybe I was clicking around too
quick. If I see it again, I'll come back to this bug.

On Thu, Jul 23, 2015 at 5:57 PM, Josh Bavari notifications@github.com
wrote:

> @cfjedimaster https://github.com/cfjedimaster do you know if the app
> crashed and you re-opened it? Hmm. Trying to figure out if it got dropped
> some how.
> 
> When you hover over the apps, a little 'x' appears. However, that prompts
> a modal asking you to confirm removing it. I'm sure that didnt show.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/driftyco/ionic-lab-issues/issues/25#issuecomment-124261020
> .

## 

# 

Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by jbavari** [comment]
Alright, thanks for the feedback. I haven't seen this myself, and I haven't been able to reproduce. Not saying it doesnt exist, just saying I cant get it to happen. Woe is me! :8ball:
